`mix rclex.prep.ros2` failed when using for Nerves development

v0.8.1 をリリースしました！のですが,,, 残念ながら問題を抱えていました,,, 
ROS 2環境無しの状態で Nerves 向けに作業を進めようとすると，`mix rclex.prep.ros2` が正常に動作しません．より適切には，このMixタスクを実行するためにはまずその前に rclex をコンパイルする必要があり，そのためには ROS 2 環境が必要となるためです．なかなか悩ましい状況です,,,

再現手順：

```
$ echo $ROS_DISTRO
(none)
$ mix nerves.new rclex_usage_on_nerves --target rpi4 
$ cd rclex_usage_on_nerves
$ export MIX_TARGET=rpi4
# add `{:rclex, "~> 0.8.1"},` to mix.exs
$ mix deps.get
$ export ROS_DISTRO=foxy

$ mix rclex.prep.ros2 
==> nerves
==> nerves_system_br
Generated nerves_system_br app
==> nerves_toolchain_ctng
Compiling 1 file (.ex)
Generated nerves_toolchain_ctng app
==> nerves_toolchain_aarch64_nerves_linux_gnu
Generated nerves_toolchain_aarch64_nerves_linux_gnu app
==> nerves_system_rpi4
Generated nerves_system_rpi4 app
==> rclex_usage_on_nerves

Nerves environment
  MIX_TARGET:   rpi4
  MIX_ENV:      dev

==> gen_state_machine
Compiling 3 files (.ex)
Generated gen_state_machine app
==> shoehorn
Compiling 7 files (.ex)
Generated shoehorn app
==> circular_buffer
Compiling 1 file (.ex)
Generated circular_buffer app
==> ring_logger
Compiling 4 files (.ex)
Generated ring_logger app
==> ssh_subsystem_fwup
Compiling 4 files (.ex)
Generated ssh_subsystem_fwup app
==> uboot_env
Compiling 5 files (.ex)
Generated uboot_env app
==> property_table
Compiling 8 files (.ex)
Generated property_table app
==> rclex
/home/takase/.nerves/artifacts/nerves_toolchain_aarch64_nerves_linux_gnu-linux_x86_64-1.6.1/bin/aarch64-nerves-linux-gnu-gcc -o /home/takase/rclex/rclex_usage_on_nerves/_build/rpi4_dev/lib/rclex/obj/msg_nif.o -c -mabi=lp64 -fstack-protector-strong -mcpu=cortex-a72 -fPIE -pie -Wl,-z,now -Wl,-z,relro -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64  -pipe -O2 --sysroot /home/takase/.nerves/artifacts/nerves_system_rpi4-portable-1.21.1/staging -g -O2 -Wall -Wextra -Wno-unused-parameter -pedantic -fPIC -I./src -I/home/takase/.nerves/artifacts/nerves_system_rpi4-portable-1.21.1/staging/usr/lib/erlang/erts-13.1.2/include -I/home/takase/.nerves/artifacts/nerves_system_rpi4-portable-1.21.1/staging/usr/lib/erlang/lib/erl_interface-5.3/include -I/home/takase/rclex/rclex_usage_on_nerves/rootfs_overlay/opt/ros/foxy/include src/msg_nif.c
src/msg_nif.c:10:10: fatal error: rmw/types.h: No such file or directory
   10 | #include "rmw/types.h"
      |          ^~~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:72: /home/takase/rclex/rclex_usage_on_nerves/_build/rpi4_dev/lib/rclex/obj/msg_nif.o] Error 1
could not compile dependency :rclex, "mix compile" failed. Errors may have been logged above. You can recompile this dependency with "mix deps.compile rclex", update it with "mix deps.update rclex" or clean it with "mix deps.clean rclex"
==> rclex_usage_on_nerves
** (Mix) Could not compile with "make" (exit status: 2).
You need to have gcc and make installed. If you are using
Ubuntu or any other Debian-based system, install the packages
"build-essential". Also install "erlang-dev" package if not
included in your Erlang/OTP version. If you're on Fedora, run
"dnf group install 'Development Tools'".

```

--- a/Makefile
+++ b/Makefile
@@ -58,7 +58,7 @@ TEMPLATES = src/msg_types_nif.h src/msg_types_nif.ec lib/rclex/msg_types_nif.ex
 calling_from_make:
 	mix compile
 
-ifneq ($(wildcard $(ROS_DIR)), "")
+ifneq ("$(wildcard $(ROS_DIR))", "")
 all: $(BUILD) $(BUILD_MSG) $(PREFIX) $(TEMPLATES) $(NIF)
 else
 all: $(TEMPLATES)

